com.civicrm.hrjobroles/tests/phpunit/CRM/Hrjobroles/BAO/HrJobRolesTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+require_once 'HrJobRolesTestBase.php';
+
+class CRM_Hrjobroles_BAO_HrJobRolesTest extends HrJobRolesTestBase {
+
+
+  function setUp() {
+    parent::setUp();
+  }
+
+  function tearDown() {
+    parent::tearDown();
+  }
+
+  function testCreateJobRoleWithBasicData() {
+    // create contact
+    $contactParams = array('first_name'=>'walter', 'last_name'=>'white');
+    $contactID = $this->individualCreate($contactParams);
+    // create contract
+    $contract = $this->createJobContract($contactID, date('Y-m-d', strtotime('-14 days')));
+    // create role
+    $roleParams = array(
+      'job_contract_id' => $contract->id,
+      'title' => 'test role'
+    );
+    $jobRole = $this->createJobRole($roleParams);
+    $this->assertNotEquals(NULL, $jobRole);
+  }
+
+  /**
+   * @expectedException PEAR_Exception
```

Comment:
I think this is a bit too generic. Maybe it would be better if you also add the @expectedExceptionMessage. Without it, any PEAR_Exception will make the test pass. Even if it's not the one you're expecting.